### ISSUE TITLE
Provide a setting to turn off the "Please review Python installation on this machine before using this extension" notification

### ISSUE THREAD

**Issue by brettcannon** [vscode-restructuredtext/vscode-restructuredtext#216]
Please follow [the troubleshooting guide](https://docs.restructuredtext.net/articles/troubleshooting.html) to locate the culprit.

If troubleshooting does not reveal any useful information, provide the following information to create a new issue,

**Environment**
- [ ] Extension version:
- [ ] Visual Studio Code version:
- [ ] Operating System version:
- [ ] Python version:
- [ ] sphinx version:
- [ ] A sample of conf.py or its important fragments:

**Expected behavior**

Not to be prompted about my Python installation when I don't need one for just syntax highlighting.

**Actual behavior**

Get notified about a Python installation missing when I actually don't want any features that require Python to be installed.

Basically I want a way to use this extension just for syntax highlighting.

**Comment by lextm** [comment]
Should be fixed in 129.0.0, but you need to disable the two preview engines via configuration,

https://github.com/vscode-restructuredtext/vscode-restructuredtext/blob/129.0.0/package.json#L334

and

https://github.com/vscode-restructuredtext/vscode-restructuredtext/blob/129.0.0/package.json#L340
